modules/ROOT/pages/db/database-connector-examples.adoc

```diff
@@ -490,6 +490,29 @@ After execution, the resulting values of the `output` and `input-output` paramet
 
 For Database Connector version 1.4.0 or later, make sure that no whitespace is between the name of the stored procedure and the first parentheses.
 
+Database connector supports the use of callable statements with the format { call procedureName(:param1, :param2, ..., :paramN) } where parameters will be matched positionally, that is to say, :paramN will match the Nth parameter in the stored procedure declaration. Suppose you have declared a stored procedure which receives two parameters: myFirstParameter and mySecondParameter, in that order. The Database connector does NOT support named parameters in callable statements, so attempting to match parameters by name ( { call myStoredProcedure(mySecondParameter => :param1, myFirstParameter => :param2) } ) will not work.
+
+It also does not support the use of engine specific embedded functions, for example, you might want to change a date to a specific format before calling a procedure in an Oracle database, however, the following approach will not work:  { call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }. All data transformations should be done using dataweave, before the invocation to the callable statement. For example, for the previous example we would perform the following transformation:
+
+[source,xml,linenums]
+----
+
+<db:stored-procedure doc:name="Stored Procedure Concatenate Date And String" config-ref="Database_Config">
+    <db:sql>{ call spConcatenateDateAndString(:name, :date, :result) }</db:sql>
+		<db:input-parameters>
+		 	<![CDATA[#[%dw 2.0
+				output application/json
+				fun format(d: DateTime) = d as String { format: "yyyy-MM-dd HH:mm:ss" }
+				---
+				{ 'date': format(|2019-10-31T13:00:00.000Z|), 'name': 'rick' }
+			]]]>
+			</db:input-parameters>
+		<db:output-parameters>
+		    <db:output-parameter key="result" type="VARCHAR"></db:output-parameter>
+		</db:output-parameters>
+	</db:stored-procedure>
+----
```

Comment:
How a customer can test this example?
Is there a way to provide the store procedure code?

Reply:
Resolved in commit [59a4c4c](https://github.com/mulesoft/docs-connectors/pull/508/commits/59a4c4c4a212175fa1612625d933c9901ebd1c9a)